assisted_compliance.md

```diff
@@ -0,0 +1,45 @@
+## Title
+Helping repo owners be compliant by writing their CONTRIBUTING.md for them as a pull request
+
+### Alternate titles
+* Let us help you be compliant
+* Assisted compliance
+
+## Problem
+The team that owns the repository doesn't have a CONTRIBUTING.md; the task force needs them to have this to submit bug fixes.
+
+## Context
+* Teams owning the repository resist compliance-related mandates for having a CONTRIBUTING.md
+* Compliance guys have to do a job; this is made difficult by teams resisting this.
```

Comment:
Can we elaborate why it is mandatory to have a contribute.md? In our case, it's mandatory to have a `readme.md` and the `contribute.md` is optional. Maybe we can abstract that somewhat.

Reply:
How about:
* Teams owning the repository resist compliance-related mandates for having a CONTRIBUTING.md. Having this file is mandated in support of the InnerSource program, to ensure that there is a known and stated process for submitting PRs and having them be appropriately checked and accepted.

Reply:
Fixed as suggested. Resolving.